src/main/java/core/basesyntax/service/action/BalanceActionHandler.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.action;
+
+public class BalanceActionHandler implements ActionHandler {
+    @Override
+    public int calculateQuantity(int currentQuantity, int inputQuantity) {
+        if (inputQuantity < 0) {
+            throw new RuntimeException("Fruit's balance can't be negative: " + inputQuantity);
+        }
+        return inputQuantity;
```

Comment:
why are you not adding to the current balance? if the fruit already has some balance